packages/fxa-content-server/webpack.config.js

```diff
@@ -52,62 +52,32 @@ const webpackConfig = {
       'node_modules',
     ],
     alias: {
-      'asmcrypto.js': path.resolve(
-        __dirname,
-        'node_modules/asmcrypto.js/asmcrypto.min.js'
+      'asmcrypto.js': require.resolve('asmcrypto.js/asmcrypto.min.js'),
```

Comment:
yarn lifts as much as it can to the root level `node_modules` to dedupe. We need to `resolve` the location instead of assuming where it is.